src/components/container/ConnectDevicesMenu/ConnectDevicesMenu.tsx

```diff
@@ -1,16 +1,16 @@
 import MyDataHelps, { DataCollectionSettings, ExternalAccount } from '@careevolution/mydatahelps-js';
-import React, { ReactElement, useEffect, useState } from 'react';
-import ConnectDevicesIcon from '../../../assets/connectdevices.svg';
-import FitbitIcon from '../../../assets/fitbit.svg';
-import GarminIcon from '../../../assets/garmin.svg';
-import AppleHealthIcon from '../../../assets/appleHealth.svg';
-import GoogleFitIcon from '../../../assets/googleFit.svg';
-import OmronIcon from '../../../assets/omron.png';
-import { Action, Title } from '../../presentational';
+import React, { ReactNode, useEffect, useState } from 'react';
+import { Action, TextBlock, Title } from '../../presentational';
 import "./ConnectDevicesMenu.css"
 import { getFitbitProviderID, getGarminProviderID, getOmronProviderID } from '../../../helpers/providerIDs';
 import { previewAccounts, previewSettings } from './ConnectDevicesMenu.previewdata';
 import language from '../../../helpers/language';
+import FitnessWearable from '../../../assets/fitness-wearable.svg';
+import FitbitLogo from '../../../assets/fitbit-logo.svg';
+import GarminLogo from '../../../assets/garmin-logo.svg';
+import AppleHealthLogo from '../../../assets/applehealth-logo.svg';
+import GoogleFitLogo from '../../../assets/googlefit-logo.svg';
+import OmronLogo from '../../../assets/omron-logo.png';
```

Comment:
This hasn't changed - I just renamed the files to be a bit more consistent.  It is SVG anyways so no need to change the files since we can just scale them - will see if they are inconsistent somehow